### PR TITLE
Spanish version or something uwu

### PR DESCRIPTION
**Pull request by Zaikonurami**
## Details

This Pull Request adds new language <Spanish>

### CODE REVIEW

**Comment by Zaikonurami** [comment]
I made bc idk, just for fun 🦊🦭

**Comment by Julli4n** [comment]
FYI due to how syncing between locales work, the credit comment will be erased when i run the workflow

File: locales/es/messages.jsonc [hunk 1]
```diff
@@ -0,0 +1,1433 @@
...
+      "homeheader_secretmessage_Default29": {
+            "message": "Hola!"
+      },
+      "homeheader_secretmessage_Default30": {
+            "message": "um... yum?"
+      },
+
+      // Added by Furrycality with Love UwU 
+      "homeheader_secretmessage_Default31": {
+            "message": "Howis!! Furrycality tradujo esto UwU"
```
Comment (juliaoverflow) [inline locales/es/messages.jsonc:569]: this wouldn't exactly work right now, but i might consider adding a secret message specifically for non-english locales as a different name, potentially `homeheader_secretmessage_NonEnglish`, i don't know

File: locales/es/messages.jsonc [hunk 2]
```diff
@@ -0,0 +1,1433 @@
...
+      "popup_extensiondata_import_alert_error": {
+            "message": "Por favor, inténtalo de nuevo con un archivo de datos válido."
+      },
+
+      "buymeacoffee": {
+            "message": "{coffeeEmoji} Comprame un café"
+      }
+}
+
+// Translated with love by Furrycality UwU - English to Spanish
```
Comment (juliaoverflow) [inline locales/es/messages.jsonc:1433]: due to how syncing between locales work, the credit comment will be erased when i run the workflow

im not sure how to properly credit at the moment

File: locales/es/messages.jsonc (hunk 1 above)
Comment (Zaikonurami) [reply locales/es/messages.jsonc:569]: I am agree

File: locales/es/messages.jsonc (hunk 2 above)
Comment (Zaikonurami) [reply locales/es/messages.jsonc:1433]: dont worry

**Comment by Zaikonurami** [comment]
Now what?

File: locales/es/messages.jsonc (hunk 2 above)
Comment (Zaikonurami) [reply locales/es/messages.jsonc:1433]: you can add a translator role in the roseal discord server, its an a idea, or make a translators credits in the credit section

**Pull request merged**